shared/src/main/java/com/faforever/neroxis/biomes/Biomes.java

```diff
@@ -20,9 +20,6 @@ public class Biomes {
     // ├-- props.json <required>
     // ├-- WaterSettings.scmwtr <required>
     // └-- Light.scmlighting <required>
-    public static final List<String> BIOMES_LIST = List.of("Brimstone", "Desert", "EarlyAutumn", "Frithen",
-                                                           "Loki", "Mars", "Moonlight", "Prayer", "Stones",
-                                                           "Syrtis", "WindingRiver", "Wonder");
     private static final String CUSTOM_BIOMES_DIR = "/custom_biome/";
 
     public static Biome loadBiome(String folderPath) {
```

Comment:
I would change this to just use the enum

Reply:
Change what exactly? The list of biomes is now in its separate enum class. Before it was here because it wasn't in a separate class, so now it is deleted from here.

Reply:
Ah, I see now you meant the string parameter. I first thought you were commenting on the deleted lines